tests/postgres_tests/migrations/0002_create_test_models.py

```diff
@@ -1,5 +1,7 @@
+from django.contrib.postgres.constraints import ExclusionConstraint
```

Comment:
could this migration adjust to initial migration?

Reply:
The migrations for `contrib.postgres` tests are split into two files, one for extensions, one for test models. There are no additional extensions that need to be enabled in the database for this constraint type to work. I don't really understand what changes you're suggesting.